dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -193,6 +196,12 @@ Configuration InstallCAM
 			DestinationPath = "$LocalDLPath\$gaAgentARM"
 		}
 
+		xRemoteFile Download_Linux_Agent_ARM
+		{
+			Uri = "$templateAgentURI/$linuxAgentARM"
```

Comment:
for all the xRemoteFiles we should be adding this as a parameter:
			MatchSource = $false

Otherwise it will re-download on reboot which we don't want.

Reply:
Ok, I will add that to all xRemoteFile